pywwt/nbextension/static/wwt_json_api.js

```diff
@@ -182,10 +182,36 @@ function wwt_apply_json_message(wwt, msg) {
       wwt.setBackgroundImageByName(msg['mode']);
       wwt.setForegroundImageByName(msg['mode']);
       break;
-
+/*
     case 'track_object':
       wwtlib.WWTControl.singleton.renderContext.set_solarSystemTrack(msg['code']);
       break;
+*/
+    case 'track_and_zoom':
+      var object = msg['obj'];
+      var ra = wwt.getRA();
+      var dec = wwt.getDec();
+      var classification = wwtlib.Classification.solarSystem;
+      var constellation = wwtlib.Place._constellation;
```

Comment:
This appears to use a private property, which is probably ok for now but it would make sense to ask whether this could be made public to avoid breakage in future?

Reply:
I wanted to include it since `constellation` is an argument for `Place.create()`, but I logged it in the console and it turns out that its value is always `undefined`. 

I think I'll do what I did with `zoom` and create the variable (so we know what's what the argument means) but not assign it a value. That way there's no need to change the visibility of any properties in the SDK.

Reply:
Sounds good!